tests/e2e_test.go

```diff
@@ -335,9 +350,20 @@ func TestPublicDBConnections(t *testing.T) {
 			wl.Deployment.Spec.Template = test.podTemplate
 			t.Log("Creating AuthProxyWorkload")
 
-			_, err = tp.CreateAuthProxyWorkload(ctx, key, appLabel, tp.ConnectionString, kind)
-			if err != nil {
-				t.Fatal(err)
+			if test.isUnixSocket {
+				p := testhelpers.NewAuthProxyWorkload(key)
```

Comment:
Should these tests also be checking that an application can reach the database? It looks like we're just checking that the pod is ready.

Reply:
Got it. I wish we could make that detail stand out some more here in the test.

Reply:
I added a comment.